dfx_extension/extension.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "azle",
+    "version": "0.21.1",
```

Comment:
please rethink the version. Is it the version of azle or does the extension have it's own version?

Reply:
I say for now let's make the version match the version of Azle. I will have this done automatically by our publish script.

The idea is that this will lead to the least amount of confusion...and hopefully we will in the future automatically install the extension with azle anyway. Right now we will proceed with manual installations though...so if the user ever wonders, they will just always install the azle extension that is the current version of azle.

We might always release the azle extension with azle anyway as well, so hopefully this will be a good long-term choice.

Reply:
The release candidate branch updated the version correctly during deploy

Reply:
I'm confused, it sounds like from the above comments we want to match the azle version but now the version has been changed to 0.0.0